docs/tutorial/connecting.txt

```diff
@@ -0,0 +1,25 @@
+=====================
+Connecting to MongoDB
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Creating a :phpclass:`Client <MongoDB\\Client>` instance
```

Comment:
The `:phpclass:` formatting here disrupts rendering of the title in the TOC panel, so I suggest just removing it.

![image](https://user-images.githubusercontent.com/244663/192605045-98a99560-e995-45eb-8aa4-a4ed071249cd.png)

```suggestion
Creating a Client instance
```